src/explore-education-statistics-admin/src/pages/release/datablocks/ReleaseDataBlockEditPage.tsx

```diff
@@ -105,12 +116,33 @@ const ReleaseDataBlockEditPage = ({
       <LoadingSpinner loading={isLoading}>
         <h2>{canUpdateRelease ? 'Edit data block' : 'View data block'}</h2>
 
-        <DataBlockSelector
-          canUpdate={canUpdateRelease}
-          publicationId={publicationId}
-          releaseId={releaseId}
-          dataBlockId={dataBlockId}
+        <FormLabel
+          id="selectedDataBlock"
+          label={
+            canUpdateRelease
+              ? 'Select a data block to edit'
+              : 'Select a data block to view'
+          }
         />
+        <div className="dfe-flex dfe-align-items--center govuk-!-margin-top-1">
+          <DataBlockSelector
+            canUpdate={canUpdateRelease}
+            publicationId={publicationId}
+            releaseId={releaseId}
+            label={false}
```

Comment:
This is the only instance where `DataBlockSelector` is used. So I don't think you need to add this extra `label` prop to `DataBlockSelector`.

The simplest solution would be to remove the label prop from DataBlockSelector and remove it from the `FormSelect` in `DataBlockSelector`. Alternatively, you could move most (if not all) these changes you've made to `ReleaseDataBlockEditPage` into `DataBlockSelector`.

In my mind, that makes sense for the label - so remove the label from the `FormSelect` in `DataBlockSelector` and move `FormLabel` into `DataBlockSelector`. You might still want to keep the `Create another data block` button in `ReleaseDataBlockEditPage`. 

But regardless happy to go along with whatever you think is best. I'm not sure if it's likely that the `DataBlockSelector` component will get used elsewhere and what labels are likely to be appropriate if it ever is used elsewhere. If it's only ever going to be used here, I'd consider even moving the above `h2` into it.